.github/workflows/test.yml

```diff
@@ -134,31 +133,9 @@ jobs:
             echo "RUSTFLAGS=-C linker=x86_64-w64-mingw32.static.posix-gcc.exe" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
             $mingw_root = "C:\rtools42\x86_64-w64-mingw32.static.posix"
             $libclang_include = "$mingw_root\include"
-          } elseif ($env:RTOOLS_VERSION -eq '40') {
-            if ($env:RUST_TARGET -like "x86_64*") {
-              $mingw_root = "C:\rtools40\mingw64"
-              $libclang_include = "$mingw_root\mingw64\x86_64-w64-mingw32\include"
-            } else {
-              $mingw_root = "C:\rtools40\mingw32"
-              $libclang_include = "$mingw_root\mingw32\i686-w64-mingw32\include"
-            }
           }
 
-          if ($env:RTOOLS_VERSION -ne '40') {
-            New-Item -Path libgcc_mock -Type Directory
-            New-Item -Path libgcc_mock\libgcc_eh.a -Type File
-            New-Item -Path libgcc_mock\libgcc_s.a -Type File
-
-            $pwd_slash = echo "${PWD}" | % {$_ -replace '\\','/'}
-            echo "LIBRARY_PATH=${pwd_slash}/libgcc_mock" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
-          }
```

Comment:
This one is needed, but without `if` now

Reply:
Thanks for catching!